evac/worker.py

```diff
@@ -257,13 +259,19 @@ def do_simulation(self):
         self.wlogger.info('Starting simulations')
         time_frame = 10
         first_evacuue = []
+        naps = 0    # how many times 
         while 1:
```

Comment:
investigate for reason of this loop

Reply:
Resolved by removing sleep function. The reason was in incomplete CFAST output. Hence the simulation should not be proceeded  - iteration results in error.